carbonmark/lib/actions.ts

```diff
@@ -277,8 +277,14 @@ export const addProjectsToAssets = async (params: {
         // project exists on API
         if (projectFromApi && projectFromApi.key) {
           project = projectFromApi;
+          // if no project on API, try subgraph
+        } else if (projectId) {
```

Comment:
The API should handle all the fallbacks ideally, and should merge in CMS data if present.

As ladytrekker said in #974 
> The creation of token contracts can be replaced with a new API endpoint which reads data from the CMS!

In other words she means we should rely on the /users endpoint for all assets and listings, and it should handle merging data and fallbacks.

Apologies for that the ticket requirements were not clear. We should deprioritize this until we have CMV2 working and then reassess.

Reply:
sounds good. I'll back burner this until after CMv2 is deployed and then will revisit.